myapp/spec/helpers/tasks_helper_spec.rb

```diff
@@ -7,33 +7,6 @@
     end
   end
 
-  describe 'value_with_i18n_line_break' do
```

Comment:
I removed these because I hid created_at column in task#index page in order to show task's tags.
When showing task's tags, it's too narrow to show created_at.

Reply:
👍